### ISSUE TITLE
[Help Wanted] Training from scratch on 1000 hours of Spanish does not work

### ISSUE THREAD

**Issue by rlenain** [SWivid/F5-TTS#565]
### Checks

- [X] This template is only for usage issues encountered.
- [X] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [X] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Environment Details

Linux, Python=3.10

### Steps to Reproduce

I ran `finetune_cli.py` with `--finetune False`, so training from scratch, with 1000 hours of Spanish data and even after 500k steps, I am still not getting speech out. It sounds like the original speaker from the prompt sometimes, but the words being uttered are complete gibberish. 

Any help on this?

### ✔️ Expected Behavior

I would like to get speech.

### ❌ Actual Behavior

Gibberish

**Comment by SWivid** [comment]
need more info, e.g. detailed configuration of training setup

**Comment by SWivid** [comment]
so actually 4*a100 `CUDA_VISIBLE_DEVICES=4,5,6,7`
thought the learning rate is too small for training from scratch
would recommend the same setting as in our paper or `train.py` (if you have pulled the latest repo, the config of base model is under config/ directory)

also you may refer to #548 , as you are using a 1000hours dataset

**Comment by rlenain** [comment]
Thank you, changing the learning rate and increasing the number of warmup updates has helped

**Comment by tuanh123789** [comment]
hi @rlenain can you share the dataset

**Comment by rlenain** [comment]
Unfortunately I cannot

**Comment by Federico1666** [comment]
> Unfortunately I cannot

puedes compartir el modelo final? estoy necesitando un buen modelo en español y el que hay disponible solo tiene 250 horas de entrenamiento

**Comment by ukemamaster** [comment]
@rlenain Is your data in LJSpeech style? Which recipe exactly did you use for your training? Can you share you training configuration?
